ansible/group_vars/dev.yml.example

```diff
@@ -1,6 +1,8 @@
+---
 project_name: dev
+project_display_name: "Software lifecycle"
+project_description: "Software Development Lifecycle (SDLC)"
```

Comment:
Good idea to add these vars to allow project display/description!

Maybe take the opportunity to customise them slightly? E.g. "Development project" vs "Staging project" or something along those lines...